presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/HashGenerationOptimizer.java

```diff
@@ -824,45 +821,47 @@ public HashComputationSet withHashComputation(Optional<HashComputation> hashComp
         }
     }
 
-    public static Optional<HashComputation> computeHash(Iterable<VariableReferenceExpression> fields)
+    public static Optional<HashComputation> computeHash(Iterable<VariableReferenceExpression> fields, FunctionManager functionManager)
```

Comment:
this could be private

Reply:
I didn't change it since it was already public but will do.